Assets/MixedRealityToolkit-Examples/UX/Models/Materials/Default.mat

```diff
@@ -0,0 +1,76 @@
+%YAML 1.1
```

Comment:
Don't we already have some sort of default material?

Reply:
Ah I see, this was an imported material. Please disable material import on the model you imported.